strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/users/UserControllerTestIT.java

```diff
@@ -444,7 +446,11 @@ void shouldBeAbleToUpdateRoles(String acceptHeader)
 
         User updatedUser = retrieveUserByName(admin.getUsername());
 
-        assertThat(SetUtils.isEqualSet(updatedUser.getRoles(), ImmutableSet.of(SystemRole.UI_MANAGER.name()))).isTrue();
+        assertThat(SetUtils.isEqualSet(updatedUser.getRoles()
```

Comment:
@anki2189 we probably shouldn't change the controller interface so it's not affect on UI.
@steve-todorov FLY

Reply:
@sbespalov `User  updatedUser` is fetched directly from DB. For UI we use `UserOutput` which is not changed. So , it should not be a problem.

Reply:
ok, thanks for clarifying